wasm2c/wasm-rt.h

```diff
@@ -245,6 +245,15 @@ typedef enum {
  */
 typedef void (*wasm_rt_function_ptr_t)(void);
 
+/**
+ * A pointer to a "tail-callee" function, called by a tail-call
+ * trampoline or by another tail-callee function. (The definition uses a
+ * single-member struct to allow a recursive definition.)
+ */
+typedef struct wasm_rt_tailcallee_t {
+  void (*fn)(void**, void*, struct wasm_rt_tailcallee_t*);
```

Comment:
Naming the params here might provide more info to the reader?

Reply:
Done.